README.md

```diff
@@ -19,6 +19,10 @@ Installing this extension will also make the latest Power Platform CLI (aka pac)
 [Power Platform CLI Exposed](https://www.youtube.com/playlist?list=PLlrxD0HtieHhEdLHxQOU96ySSZpMCyAxf)
 
 ## Release Notes
+2.0.0:
+  - pac CLI 1.25.1 (June refresh, see release notes on [nuget.org](https://www.nuget.org/packages/Microsoft.PowerApps.CLI/))
+  - Multi file editing support for Power Pages components in VS Code for Web
```

Comment:
Placeholder line for now;  Power Pages team, please reply to comment here with what you want added to the release notes

Reply:
Power Pages site code edit support in VScode web extension for components like forms, content snippets, lists, and more.

Reply:
updated